### DISCUSSION TITLE
german language models in version 3.2.0 not found as conda package

### DISCUSSION THREAD

**Discussion by chriga-arch** [explosion/spaCy#9788] [Help: Installation] [answered]
## How to reproduce the behaviour
Create conda environment in form of environment.yml like the following:

```
channels:
  - conda-forge
dependencies:
  - python=3.7
  - numpy=1.21.2
  - pip=21.2.4
  - scikit-learn
  - scipy=1.7.1
  - pandas
  - spacy
  - spacy-lookups-data
  - spacy-model-de_core_news_lg=3.2.0

```
## Your Environment
* Operating System: Linux
* Python Version Used: 3.7
* spaCy Version Used: 3.2.0
* Environment Information: Conda (channel: conda-forge)

**Comment by polm** [comment, marked as answer]
Sorry this is confusing. There are some spaCy models on conda-forge, but we did not upload those, they were added by a user at some point and we don't maintain them. We recommend you follow the [install instructions](https://spacy.io/usage#quickstart) to install spaCy. 

In a requirements.txt file, it's possible to provide a URL, such as those spaCy uses for models. I'm not sure if there's an equivalent feature in conda.

**Reply by chriga-arch** [reply]
Okay thanks for the infos. Sorry i thought the packages on channel conda-forge are managed and maintained by the official publisher. 

Tried it with requirements.txt file as you mentioned - works! Thanks a lot!
